src/executables/csql_result_format.c

```diff
@@ -1938,6 +1958,27 @@ csql_db_value_as_string (DB_VALUE * value, int *length, bool plain_string, CSQL_
       }
       break;
 
+    case DB_TYPE_MIDXKEY:
+      {
+	if (csql_arg->midxkey_print)
+	  {
+	    string_buffer sb;
+
+	    sb.clear ();
+	    db_sprint_value (value, sb);
+
+	    /*  Make sure it has the same format as the result of using cast().
+	     *  '{1, 'abc'}'  or  '{1, ''abc''}'  (depends on change_single_quote)
+	     */
+	    result =
```

Comment:
doesn't it needs to check if it's null?

Reply:
What specifically do you think requires a NULL check?